yarn.lock

```diff
@@ -364,9 +364,9 @@
     tslib "^1.8.0"
```

Comment:
This file should not have changed, as we did not bump any dependencies. We are still using `version "1.0.0-rc.7"` for the dynamo SDK. This is weird..

Reply:
hmmm, i did reinstall the `yarn.lock` following malfunctions with `taico`.

First I ran `yarn` after a rebase to add missing dependencies
Then I started to have problems with `e2e` tests
I deleted the `yarn.lock` file and I ran `yarn` once again without changing the `package.json`

This is all I did